tests/unit/transformers/test_categorical.py

```diff
@@ -32,10 +32,24 @@ def test__get_intervals(self):
 
         # Asserts
         expected_intervals = {
-            'foo': (0, 0.5, 0.25, 0.5 / 6),
-            'bar': (0.5, 0.8333333333333333, 0.6666666666666666, 0.05555555555555555),
-            'tar': (0.8333333333333333, 0.9999999999999999, 0.9166666666666666,
-                    0.027777777777777776)
+            'foo': (
```

Comment:
Would the original `'foo': (0, 0.5, 0.25, 0.5 / 6),` be valid, since it opens and closes on same line? It seems a bit excessive to line break after every element here, since it fits neatly onto one line.

Reply:
It has to follow the same multi-line for the containers:
```python
        expected_intervals = {
            'foo': (0, 0.5, 0.25, 0.5 / 6),
            'bar': (0.5, 0.8333333333333333, 0.6666666666666666, 0.05555555555555555),
            'tar': (0.8333333333333333, 0.9999999999999999, 0.9166666666666666,
                    0.027777777777777776)  # this is different from the others, so it becomes invalid.
        }
``` 
If it was like this it would be valid:
```python
        expected_intervals = {
            'foo': (0, 0.5, 0.25, 0.5 / 6),
            'bar': (0.5, 0.8333333333333333, 0.6666666666666666, 0.05555555555555555),
            'tar': (0.8333333333333333, 0.9999999999999999, 0.9166666666666666, 0.027777)
        }

```

Reply:
Sounds good, thanks for the update.

Would prefer if a mixed style was allowed, but I'm fine with trying out the consistent styling requirements.